src/NUnitFramework/tests/Internal/EventQueueTests.cs

```diff
@@ -145,7 +145,7 @@ public class SetWaitHandle_Enqueue_AsynchronousTest : ProducerConsumerTest
 #endif
             public void SetWaitHandle_Enqueue_Asynchronous()
             {
-                using (AutoResetEvent waitHandle = new AutoResetEvent(false))
+                using (new AutoResetEvent(false))
```

Comment:
Why create `AutoResetEvent` at all?

Reply:
no idea. it was added by @CharliePoole 9 years ago https://github.com/nunit/nunit/commit/4bbf615e01bc0d6078eac6dce07d4f0aac5c03ec

Reply:
seems the handle was previously passed into `SetWaitHandleForSynchronizedEvents` https://github.com/nunit/nunit/blame/024bab1a77aee1e56a8851ac3bf5997c578071be/NUnitFramework/src/tests/Internal/EventQueueTests.cs#L208-L215

i assume that functionality has been refactored away and this test is no longer relevant?

Reply:
I think you may be correct.

This historical comment here implies it was used for synchronous events: https://github.com/nunit/nunit/blob/96cb0664cd505beaddfdf0e300c1dc82951e2139/src/NUnitFramework/framework/Internal/Execution/EventQueue.cs#L99

Which were removed (also by @CharliePoole ) in https://github.com/nunit/nunit/pull/1526

Reply:
That sounds accurate.

Reply:
test removed